tests/src/Kernel/ViewsTableTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel;
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Core\Database\Database;
+use Drupal\Tests\oe_bootstrap_theme\Kernel\Traits\RenderTrait;
+use Drupal\views\Tests\ViewTestData;
+
+/**
+ * Tests rendering of the pager.
+ */
+class ViewsTableTest extends AbstractKernelTestBase {
+
+  use RenderTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'system',
+    'views',
+    'views_test_config',
+    'views_test_data',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Configuration and schemas needed for view test.
```

Comment:
instead of this comment and the one from line 38, rather explain why we do all this code ex: `// Here we set up tralalal` and leave a @see reference to ViewsKernelTestBase::setUpFixtures()

Reply:
Ok, this way explains why the code it's there and that I'm following core